app/StateMachines/SiteStatusStateMachine.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\StateMachines;
+
+class SiteStatusStateMachine extends EntityStatusStateMachine
+{
+    public const DRAFT = 'draft';
+    public const AWAITING_APPROVAL = 'awaiting-approval';
+    public const APPROVED = 'approved';
+    public const NEEDS_MORE_INFORMATION = 'needs-more-information';
+    public const RESTORATION_IN_PROGRESS = 'restoration-in-progress';
+
+    public function recordHistory(): bool
+    {
+        return true;
+    }
```

Comment:
There's no reason to override this method with the same value already set in the super class.

Reply:
removed